task3/script.js

```diff
@@ -0,0 +1,15 @@
+const ul = document.getElementById('ul');
+
+for (let i =1; i <=2; i++) {
+    const li =document.createElement('li');
+    const a =document.createElement('a');
+    const img =document.createElement('img');
+
+    a.href = "a" + i + ".html";
+    a.textContent = "a" + i;
```

Comment:
[IMO]

>  a.href = "a" + i + ".html";
>  a.textContent = "a" + i;

It will work if you write it this way, but you can also use the
Using the template literal method, it is also possible to embed variables directly in the string.

Reply:
I didn't realize that using template literals would make it easier to understand.

I understand.
I'll correct it

Reply:
Good!
You have a good command of template literals.